atlasdb-api/src/main/java/com/palantir/atlasdb/factory/TransactionManagerConsistencyResult.java

```diff
@@ -23,7 +23,9 @@
  * determine that it is safe (or, at least, not obviously unsafe) to service requests.
  */
 @Value.Immutable
+@SuppressWarnings("ClassInitializationDeadlock")
```

Comment:
```suggestion
@SuppressWarnings("ClassInitializationDeadlock")
@Value.Style(visibility = Value.Style.ImplementationVisibility.PACKAGE)
```

Reply:
And add a builder if necessary for outside use, the idea is not to expose the generated immutable

Reply:
Curious why suppress an error-prone check for potential class initialization deadlock (see https://github.com/palantir/gradle-baseline/pull/1598 that @carterkozak just added )? I'm assuming this is a fairly common pattern in many uses of immutables where there's either some constant defined as the empty object or similar. Do we need to consider pulling these types of constants out to a separate class (e.g. `TransactionManagerConsistencyResults`) or lazy initialization get of a memoized supplier?